docs/content/components/hero.mdx

```diff
@@ -207,12 +213,71 @@ code_snippets:
           <div class="nci-hero__nci-cta-strip-container">
             <div class="grid-container">
               <ul class="nci-cta-strip">
-                <li><a href="#!" class="usa-button">Programs</a></li>
+                <li><a href="http://www.cancer.gov" class="usa-button">Programs</a></li>
+                <li>
+                  <a href="http://www.cancer.gov" class="usa-button">Resources</a>
+                </li>
+                <li>
+                  <a href="http://www.cancer.gov" class="usa-button">Success Stories</a>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </div>
+    - title: Default Hero with Button & Text Box and CTA Strip External Links
+      preview: true
+      code: |
+        <div class="nci-hero nci-hero--with-cta-strip">
+          <picture class="nci-hero__image">
+            <source
+              media="(min-width: 1024px)"
+              srcset="../nci-hero-images/hero-widescreen.jpg"
+            />
+            <source
+              media="(min-width: 880px)"
+              srcset="../nci-hero-images/hero-desktop.jpg"
+            />
+            <source
+              media="(min-width: 640px)"
+              srcset="../nci-hero-images/hero-tablet-large.jpg"
+            />
+            <source
+              media="(min-width: 480px)"
+              srcset="../nci-hero-images/hero-tablet.jpg"
+            />
+            <source
+              media="(min-width: 320px)"
+              srcset="../nci-hero-images/hero-mobile-large.jpg"
+            />
+            <img src="../nci-hero-images/hero-mobile.jpg" alt="" />
+          </picture>
+          <div class="nci-hero__cta-container">
+            <div class="nci-hero__cta nci-hero__cta--with-button">
+              <h2 class="nci-hero__cta-tagline">
+                NCI is the nation's leader in cancer research.
+              </h2>
+              <a href="#!" class="nci-hero__cta-button usa-button">
+                Learn More
+              </a>
+            </div>
+          </div>
+          <div class="nci-hero__nci-cta-strip-container">
+            <div class="grid-container">
+              <ul class="nci-cta-strip">
+                <li>
+                  <a href="http://www.google.com" class="usa-button">
+                  Link to Google
+                  </a>
+                </li>
                 <li>
-                  <a href="#!" class="usa-button">Resources</a>
+                  <a href="http://www.google.com" data-ncids-internal-link class="usa-button">
+                  Override External Link Icon
+                  </a>
                 </li>
                 <li>
-                  <a href="#!" class="usa-button">Success Stories</a>
+                  <a href="http://www.cancer.gov" class="usa-button ncids-external-link">
```

Comment:
we should probably use something like `nci-cta-strip--external` rather than `usa-button--external` so that we can mark the component external as a whole, and then target the `usa-button` appropriately as a child of the external class instead

Reply:
my only concern here is that cta has three buttons, whereas promo block has one. CTA could only require one button needs external link icon - So placement of that modifier at the component level might be bad for CMS side.

However the external link mixin is inside of the CTA strip class so the selector is only targeting usa-button inside of cta-strip currently.
```ul.nci-cta-strip li .usa-button:not(.usa-button--external):not([href*=".gov"], [href*="mailto:"], [href*="javascript:"], [href*="public.govdelivery.com"], [href*="www.gov.uk"], [data-ncids-internal-link])..```

Reply:
disregard this comment as there are several links in the strip, don't know if that would work for this one.